java/maven/src/org/netbeans/modules/maven/execute/MavenCommandLineExecutor.java

```diff
@@ -191,6 +198,7 @@ public ExecutorTask execute(RunConfig config, InputOutput io, TabContext tc) {
     public MavenCommandLineExecutor(RunConfig conf, InputOutput io, TabContext tc) {
         super(conf, tc);
         this.io = io;
+        this.trace = new Throwable();
```

Comment:
nitpick: maybe wrap in `if (LOGGER.isLoggable(FINE))`?

throwable creation can become annoying when JFR is active since this will [show up as event](https://github.com/openjdk/jdk/blob/5f357fa27d89a3ead3783a3197ba4c576802cb7a/src/java.base/share/classes/java/lang/Throwable.java#L265-L267) in the JFR recordings, while the typical expectation is to see "real" exceptions.